MapboxCoreNavigation/CLLocationDirections.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import CoreLocation
+
+extension CLLocationDirection {
+    func clockwiseDifference(from toDirection: CLLocationDirection) -> CLLocationDirection {
```

Comment:
It’s a bit confusing to have an argument called “from” externally but “to” internally. Let’s call this angle `otherDirection`.